Asus/ZenFoneMaxM2/res/xml/power_profile.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="screen.on">63</item>
+    <item name="screen.full">261</item>
+    <array name="cpu.speeds">
```

Comment:
No modem? bluetooth? Wifi?

Reply:
This was durectly extracted from framework-res and other framework files